netbeans.apache.org/src/content/kb/docs/ide/git.asciidoc

```diff
@@ -35,8 +35,6 @@ Git is a free and open source, distributed version control system designed to ha
 
 === Contents
 
-image::images/netbeans-stamp-80-74.png[title="Content on this page applies to the NetBeans IDE 7.4 and 8.0"]
-
 * <<initialize,Initializing a Git Repository>>
```

Comment:
This is the list of entries that should be removed, as these will be generated automatically.